### ISSUE TITLE
Open Source Licensing

### ISSUE THREAD

**Issue by peculiaruc** [chaoss/education#95]
Description:
Open Source Licensing: Give an overview of different open-source licenses, their implications and how to choose the right one for your project.

 Write a Script - Check out the[ samples ](https://github.com/chaoss/education/blob/main/Submission.md)and use this [template](https://docs.google.com/document/d/1jIee4PRQsjF4_nDBxNOcbAxeQ1WA0w5HKOlRc3wdR34/edit)
Create a Slide deck - Find a sample [here](https://github.com/chaoss/education/blob/main/Submission.md), and use this [template](https://docs.google.com/presentation/d/1t_oBQhMAtZvvsqTppRj96fbjpokakjiZOdtWIO3NIdY/edit#slide=id.g224756f3bb6_1_0)
Record a video - [Sample](https://github.com/chaoss/education/blob/main/Submission.md) 

**Additional context**

- Kindly go through the [ReadMe.md](https://github.com/chaoss/education/blob/main/README.md) file. 
- [Contributing.md ](https://github.com/chaoss/education/blob/main/Contributing.md)file contains information that will help you get started
- Choose your preferred [background](https://github.com/chaoss/education/blob/main/README.md#video-background) for your video recording.

**Comment by Aysha-py** [comment]
Hi Peculiar, I would love to work on this.

**Comment by peculiaruc** [comment]
@Aysha-py How is this coming on?

**Comment by Aysha-py** [comment]
@peculiaruc It coming on great. I already started the Script. I will turn in tomorrow for review and feedback.

**Comment by peculiaruc** [comment]
> @peculiaruc It coming on great. I already started the Script. I will turn in tomorrow for review and feedback.

Awesome!

**Comment by Aysha-py** [comment]
Hi @peculiaruc , I added the script at #122. Kindly review and provide feedback. Thank you.

**Comment by jwflory** [comment]
I reviewed the [script](https://docs.google.com/document/d/17UkQs5wJE9ZDJLySaL9O2bqrRNndztkrF30Yi71iqsU/edit?usp=sharing) for this topic. It looks good to me! I saw some parts that presented copyleft licenses as anti-commercial, which could be true in some cases but actually can be pro-business in other cases too.

I was wondering what the status update for this course is? If I wanted to contribute, is it possible?

**Comment by peculiaruc** [comment]
> I reviewed the [script](https://docs.google.com/document/d/17UkQs5wJE9ZDJLySaL9O2bqrRNndztkrF30Yi71iqsU/edit?usp=sharing) for this topic. It looks good to me! I saw some parts that presented copyleft licenses as anti-commercial, which could be true in some cases but actually can be pro-business in other cases too.
> 
> I was wondering what the status update for this course is? If I wanted to contribute, is it possible?

Yes it is possible. Since the contributor @Aysha-py is yet to record the video, your input is welcome.

**Comment by peculiaruc** [comment]
> I reviewed the [script](https://docs.google.com/document/d/17UkQs5wJE9ZDJLySaL9O2bqrRNndztkrF30Yi71iqsU/edit?usp=sharing) for this topic. It looks good to me! I saw some parts that presented copyleft licenses as anti-commercial, which could be true in some cases but actually can be pro-business in other cases too.
> 
> I was wondering what the status update for this course is? If I wanted to contribute, is it possible?

Hi @jwflory  We would like to get your input on this issue, as you mentioned earlier, before proceeding to video recording.

**Comment by peculiaruc** [comment]
Thank you @Aysha-py  for working on this course
